### ISSUE TITLE
How to set attention mask, any suggestion?

### ISSUE THREAD

**Issue by thesby** [allenai/longformer#150]
As mentioned by `https://github.com/allenai/longformer/issues/75#issuecomment-651295800` , the default is only local attention. So how to generate good global attention? any suggestions about it?

I find a example about it in `README.md`
```
attention_mask = torch.ones(input_ids.shape, dtype=torch.long, device=input_ids.device) # initialize to local attention
attention_mask[:, [1, 4, 21,]] =  2  # Set global attention based on the task. For example,
                                     # classification: the <s> token
                                     # QA: question tokens
```
Why to set `1, 4, 21` global attention ?

**Comment by ibeltagy** [comment]
`1, 4, 21` are just random locations for the code example. Global attention should be set in a task-specific way taking the task details into account.

**Comment by pratikchhapolika** [comment]
> `1, 4, 21` are just random locations for the code example. Global attention should be set in a task-specific way taking the task details into account.

The document or code snippet about  document embedding is not very clear.  For simple sentences..right where <s> and </s> are start and end of documents, how should we calculate the embeddings? Should we give global attention=2 to starting  and end  of every documents and pass global attention as a parameter to model?

**Comment by pratikchhapolika** [comment]
> `1, 4, 21` are just random locations for the code example. Global attention should be set in a task-specific way taking the task details into account.

The document or code snippet about  document embedding is not very clear.  For simple sentences..right where `<s>` and `</s>` are start and end of documents, how should we calculate the embeddings? Should we give global attention=2 to starting  and end  of every documents and pass global attention as a parameter to model?
